RTRTextures.vcxproj

```diff
@@ -1,245 +1,234 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Not sure whether it would be a good idea to migrate projects to vs2017 as most of the people might not have it.
Any motivations about this change?

Reply:
Since present contributors have vs2017 i'm fine with this.  (confirmed with Rohit and Rupesh)